src/utilities/parseRelayConnectionArguments.ts

```diff
@@ -0,0 +1,91 @@
+import { GraphQLError } from "graphql";
```

Comment:
delete this file, and use the function exported from `src/utilities/graphQLConnection/parseGraphQLConnectionArguments.ts`